testutil/TestUtil.cs

```diff
@@ -77,9 +77,9 @@ public T Eventually<T>(Func<T> func)
                 catch (Exception e) when (ShouldCatch(e) && i < MaxTryCount)
                 {
                     int jitteredDelayMs;
-                    lock(_lock)
+                    lock (_lock)
```

Comment:
Please revert these changes, although they are good changes, it's best to keep this PR clean of unrelated changes. Feel free to push a new PR with these.

Reply:
Reverted